src/commands/fight.js

```diff
@@ -124,12 +128,19 @@ function fight(message, player1, player2, turn) {
 
     }).catch(() => {
         message.channel.send(`**${currentPlayer.user.username}** didn't respond, so we can assume that he has lost his turn.`);
+        currentPlayer.miss++;
         fight(message, player1, player2, !turn);
+        if (currentPlayer.miss === 2) {
```

Comment:
Just in case of something odd happening, it would better to check if it's `>= 2` instead of `=== 2`.